<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,10,1,1">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1,6,1,1">
      <item>
       <widget class="QPushButton" name="open_dir_btn">
        <property name="text">
         <string>Open Dir</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="open_files_btn">
        <property name="text">
         <string>Open Files</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="mode_label">
        <property name="text">
         <string>Current Processing Mode:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="processing_mode_btn">
        <property name="text">
         <string>Copy</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="fd_info_label">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="img_show_label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="type3_btn">
        <property name="text">
         <string>aslb</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">type_btn_group</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="type2_btn">
        <property name="text">
         <string>arbx</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">type_btn_group</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="type1_btn">
        <property name="text">
         <string>abox</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">type_btn_group</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="type4_btn">
        <property name="text">
         <string>aslw</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">type_btn_group</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="8,1,1">
      <item>
       <widget class="QLabel" name="processed_info_label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="undo_btn">
        <property name="text">
         <string>Undo</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="type_btn_group"/>
 </buttongroups>
</ui>
